### DISCUSSION TITLE
How to set a Limit in custom field (text, booking related)

### DISCUSSION THREAD

**Discussion by Apfelkomplott** [liedekef/events-made-easy#97] [Q&A] [answered]
Hi Franky.
I’m the guy using your script with the garage-sales, perhaps you remember.
Going into the third year using eme, I now force some descriptions of the offer, while registering to the event.
Like “old vinyls, a bobby car, blankets aso”.
How can I setup this field (named offer) for a maximum input length of ie. 250 characters?
Thanks in advance!

Regards from Germany 
Alex.

**Comment by liedekef** [comment, marked as answer]
When defining a field, you can set also "HTML field attributes". In your case, I guess maxlength=250 will do.
